src/view/accessToken.js

```diff
@@ -0,0 +1,120 @@
+module.exports = token => `<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/combine/npm/@zendeskgarden/css-utilities@3.0.2,npm/@zendeskgarden/css-tooltips@4.0.5,npm/@zendeskgarden/css-tags@4.0.5,npm/@zendeskgarden/css-tabs@5.0.5,npm/@zendeskgarden/css-tables@3.0.7,npm/@zendeskgarden/css-pagination@3.0.5,npm/@zendeskgarden/css-modals@6.2.2,npm/@zendeskgarden/css-menus@7.0.6,npm/@zendeskgarden/css-grid@0.1.8,npm/@zendeskgarden/css-forms@7.0.19,npm/@zendeskgarden/css-chrome@3.0.7,npm/@zendeskgarden/css-callouts@3.3.16,npm/@zendeskgarden/css-buttons@6.0.5,npm/@zendeskgarden/css-bedrock@7.0.5,npm/@zendeskgarden/css-avatars@3.0.5,npm/@zendeskgarden/css-arrows@3.1.1">
+    <title>agnoStack: Shopify Access Token</title>
+    <style>
+      .c-btn--xs {
+        display: flex;
+        justify-content: center;
+        min-width: 0;
+        padding: 0 12px;
+        line-height: 20px;
+        border-radius: 4px;
+        margin-left: 4px;
+      }
+
+      .c-txt__input--buttonxs {
+        padding: 4px 6px;
+        min-height: unset;
+        align-items: center;
+      }
+
+      .container--center {
+        height: 90vh;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+      }
+
+      .c-tooltip--light--copied {
+        position: absolute;
+        padding: 4px;
+        top: 3px;
+        right: -130px;
+        z-index: 1;
+      }
+
+      .c-tooltip--hide {
+        visibility: hidden;
+        transition: ease-in;
+      }
+
+      .row--margin {
+        margin: 20px 0;
+      }
+
+      #logo {
+        display: flex;
+        justify-content: center;
+      }
+      #logo img {
+        width: 15vw;
+        max-width: 175px;
+      }
+    </style>
+    <script>
+      function copy() {
+        const copyText = document.getElementById('accessToken')
+        const tooltip = document.getElementsByClassName('c-tooltip--light')[0]
+
+        copyText.select();
+        copyText.setSelectionRange(0, 99999)
+
+        document.execCommand("copy");
```

Comment:
need to wrap this in 
```
    if (document.execCommand) {
```

Reply:
and switch to single quotes around `execCommand('copy')`

Reply:
May need to look at `document.queryCommandSupported`??

Reply:
Something similar to this??
```
    if (document.execCommand && document.queryCommandSupported('copy') && codeRef && codeRef.current && codeRef.current.value) {
      // 1.) does copy empty inputs, but adds newline before content
      const range = document.createRange();
      range.selectNode(codeRef.current);
      window.getSelection().removeAllRanges();
      window.getSelection().addRange(range);
      
      // 2.) doesn't copy empty inputs
      codeRef.current.select();
      
      // 3.) doesn't copy empty inputs
      codeRef.current.focus();
      document.execCommand('selectall');

      document.execCommand('copy');
    }
```

Reply:
The function copy has been changed as below.

```
function copy() {
    const copyText = document.getElementById('accessToken');
    const tooltip = document.getElementsByClassName('c-tooltip--light')[0];

    if (document.execCommand && document.queryCommandSupported('copy') && copyText && copyText.value) {
      const range = document.createRange();
      range.selectNode(copyText);
      window.getSelection().removeAllRanges();
      window.getSelection().addRange(range);
      
      copyText.select();
      
      copyText.focus();
      document.execCommand('selectall');

      document.execCommand('copy');
    }

    tooltip.classList.remove('c-tooltip--hide');
    setTimeout(function() {
      tooltip.classList.add('c-tooltip--hide');
    }, 500)
  }
```

Reply:
Did you test it @parigu2??